### ISSUE TITLE
Mac Chrome push notification support?

### ISSUE THREAD

**Issue by jaminroe** [needim/noty#421]
Was looking at the [supported browsers/OS table](https://github.com/needim/noty/blob/master/docs/browsers.md), and noticed that Chrome for Mac wasn't listed. Should it be listed?

**Comment by needim** [comment]
Yeah, should. Sorry about that. I will update the table for supported push notification feature. Firefox supported too, but Safari, not atm.

**Comment by jaminroe** [comment]
Cool no problem! 👍 Sounds good 😊
